models/node_helpers.go

```diff
@@ -161,12 +162,6 @@ func createNodeWithID(q *reform.Querier, id string, nodeType NodeType, params *C
 		}
 	}
 
-	if params.Region != nil {
```

Comment:
this check still should be here, just without reregister logic

Reply:
Logic has been moved here: https://github.com/percona/pmm-managed/blob/PMM-4919_add_force_not_working/models/node_helpers.go#L69-L71